### ISSUE TITLE
Taxon constraint:  cell death related taxon constraints

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#24653]
Please provide as much information as you can: 

* **GO term ID label:**

GO:1902445  regulation of mitochondrial membrane permeability involved in programmed necrotic cell death
~GO:0097300  programmed necrotic cell death~ ignore
~GO:0008219 cell death~ obsoleted 
~GO:0070265   necrotic cell death~ obsoleted

* **Request to add a taxon constraint:**

* * **never in taxon: ** 
never in yeast (@srengel do you agree?)

* **Supporting evidence if available (e.g PMID):**

**Comment by ValWood** [comment]
alerted fom

https://github.com/geneontology/go-annotation/issues/4395

**Comment by srengel** [comment]
wait, how did we go from removing the bad annotations to now saying 'never in yeast'?

we're not comfortable saying 'never in yeast', as necrosis seems to happen in S.cerevisiae.  Edith and Rob dug up some papers:

**Comment by ValWood** [comment]
Apologies. I listed 
GO:0097300   programmed necrotic cell death
instead of "cell death"
(now edited)

GO:0097300   programmed necrotic cell death was never intended for obsoletion. 

I was not aware that yeast used "necrotic cell death" as a method for population survival so I do not need to add this taxon restriction. 

However, I looked at the papers above and these don't seem to describe a programmed necrotic pathway (the earlier paper seems to be mutants which increase viability in stationary phase when high concentrations of glucose are added after glucose depletion?)
